### DISCUSSION TITLE
<search in topics>

### DISCUSSION THREAD

**Discussion by pizzav-xyz** [community/community#128977] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

how do i search in [topics](https://github.com/topics/dotfiles)

**Comment by jzelAdmin2006** [comment, marked as answer]
Hi @kittenvr

Thank you for your question.
You can search for topics by using the general search functionality at the top on the right. After that, you can select the filter [Topics], which will automatically add the URL parameter "&type=topics".

![image](https://github.com/community/community/assets/112761658/231a5e14-1fe5-495e-87a3-e1a17c4ded00)
![image](https://github.com/community/community/assets/112761658/577f1a77-4f88-4661-85ab-06eb0bd06654)
![image](https://github.com/community/community/assets/112761658/89185447-1027-4bb3-b840-bde89b284bc3)

Once you've found your topic, you can again use the general search field and use the prefix "topic:" with the complete name of your topic directly after that. Your search query can simply be typed put after a space. This will allow you to search within a certain topic. Of course you can select a filter e.g. in order to search for repos (although for the "topic" qualifier only repos and packages are supported).

![image](https://github.com/community/community/assets/112761658/617ebde0-ec87-4370-a9c9-8fd7805195a0)


I hope this will help you. If that's the case, please make sure to mark my comment as the answer.

All the best
JZEL

**Reply by pizzav-xyz** [reply]
how do i make it so only repos that got updated this year get show?

**Reply by jzelAdmin2006** [reply]
You can add the qualifier "pushed" to your search.

![image](https://github.com/community/community/assets/112761658/31650344-1850-4115-9c99-9f59644ed727)
